site/pages/competencies/how-to-use.md

```diff
@@ -26,7 +26,29 @@ So, for example, a Junior Engineer should be working towards the competencies at
 
 Each competency has a summary which is designed to prompt a yes/no response. For each competency, an engineer should feel able to answer either _"yes, I'm meeting this competency"_, or _"no, I'm not meeting this competency yet"_.
 
-Engineers are expected to meet all of the competencies for a level in order to be promoted, _except_ for domain-specific competencies that don't apply to their role or specialism.
+In order to be promoted, an engineer must specify which competencies they are meeting for a level. We don't expect that an engineer will _meet_ all of the competencies, but the promotions board will consider all of the competencies when making a decision. Domain-specific competencies that don't apply to an engineer's role or specialism will not be considered as part of a promotions case.
```

Comment:
```suggestion
In order to be promoted, an engineer must specify which competencies they are meeting for a level. We don't expect that _every_ engineer will meet _all_ of the competencies, but the promotions board will consider all of the competencies when making a decision. Domain-specific competencies that don't apply to an engineer's role or specialism will not be considered as part of a promotions case.
```

Something about the sentence was niggling me. I think it's that `an` might be a weird choice of cardinality given the intent of the sentence. And maybe I'm wrong, but feels more natural to emphasise `all` because emphasising `meet` kind of begs the question 'Well what is there other than `meet`? `game`? `fudge`?'.  With `all` the implication that we mean `most` feels clearer